doc/Security.xml

```diff
@@ -866,7 +653,7 @@
         </section>
         <section>
           <title> Revocation Status Checking </title>
-          <para>By default, a device shall use the parameter values defined in <xref linkend="_Ref395170420"/> for the parameters defined in RFC 5280, Sect. 6.1.1 for the CRL-based certificate revocation status checking algorithm defined in Sect. <xref linkend="_Ref392600406"/>.</para>
+          <para>By default, a device shall use the parameter values defined in <xref linkend="_Ref395170420"/> for the parameters defined in RFC 5280, Sect. 6.1.1.</para>
```

Comment:
Suggestion to improve this as below with correction section numbers. 
`By default, a device shall use the parameter values defined in <xref linkend="_Ref395170420"/> for the parameters specified in RFC 5280, Sect. 6.3.1, for the CRL processing outlined in RFC 5280, Sect. 6.3.3.`